pilot/pkg/proxy/envoy/testdata/cds-ssl-context.json.golden

```diff
@@ -175,13 +175,7 @@
      {
       "url": "tcp://httpbin.default.svc.cluster.local:80"
      }
-    ],
-    "ssl_context": {
-     "cert_chain_file": "/etc/certs/cert-chain.pem",
-     "private_key_file": "/etc/certs/key.pem",
-     "ca_cert_file": "/etc/certs/root-cert.pem",
-     "verify_subject_alt_name": []
-    }
+    ]
```

Comment:
Why is the context removed ?

Reply:
because its wrong.. it should not be there. Thats the bug fix :)

Reply:
@costinm httpbin is a Kubernetes ExternalName service - it does not have Envoy proxy attached, so mTLS should not be here.